src/shravani_rapelli/Assignment79.java

```diff
@@ -0,0 +1,35 @@
+package shravani_rapelli;
+
+public class Assignment79 {
+
+	void numberPatternPriting2() {
+		int temp=4;
```

Comment:
Write generic code so that this pattern will print as per user input like 6 rows or 2 rows